src/config.c

```diff
@@ -283,7 +286,12 @@ int config_parse_conf(struct config *cfg, const struct conf *conf)
 	(void)conf_get_bool(conf, "rtcp_mux", &cfg->avt.rtcp_mux);
 	(void)conf_get_range(conf, "jitter_buffer_delay",
 			     &cfg->avt.jbuf_del);
+
+	/* QoS Settings */
 	(void)conf_get_bool(conf, "rtp_stats", &cfg->avt.rtp_stats);
+	(void)conf_get_bool(conf, "rtcpxr_stats", &cfg->avt.rtcpxr_stats);
+	(void)conf_get_str(conf, "rtcpxr_collector",
+	      cfg->avt.rtcpxr_collector, sizeof(cfg->avt.rtcpxr_collector));
```

Comment:
does the rtcpxr_stats config require rtcpxr_collector to be set?

the rtcpxr_collector string should be parsed and validated here.